Source/Combine.swift

```diff
@@ -22,7 +22,7 @@
 //  THE SOFTWARE.
 //
 
-#if canImport(Combine)
+#if !(os(iOS) && (arch(i386) || arch(arm)))
```

Comment:
this check must be broader to check for windows and linux that doesn't support Combine framework